client/blocks/reader-author-link/index.jsx

```diff
@@ -9,7 +9,7 @@ import classnames from 'classnames';
 /**
  * Internal dependencies
  */
-import { isAuthorNameBlacklisted } from 'reader/lib/author-name-blacklist';
+import { isAuthorNameBlocklisted } from 'reader/lib/author-name-blocklist';
```

Comment:
Just a not, but perhaps `isAuthorNameBlocked`? `is...Blocklisted` reads a little odd to me. Either is better than what we have though, so don't let it hold things up.

Reply:
Good shout - `isAuthorNameBlocked` is much clearer 👍